root/components/FormRowRadio.js

```diff
@@ -30,7 +30,6 @@ type Props = {|
 const FormRowRadio = ({
   field,
   label,
-  onChange,
```

Comment:
Unused prop (which doesn't appear in the `Props` type, either), which Flow now doesn't like.